src/apps/backend/modules/communication/internals/twilio_service.py

```diff
@@ -0,0 +1,41 @@
+from typing import Optional
+
+from twilio.base.exceptions import TwilioException
+from twilio.rest import Client
+
+from modules.communication.errors import ServiceError
+from modules.communication.internals.twilio_params import SMSParams
+from modules.communication.types import SendSMSParams
+from modules.config.config_service import ConfigService
+
+
+class TwilioService:
+    __client: Optional[Client] = None
+
+    @staticmethod
+    def send_sms(params: SendSMSParams) -> None:
+        SMSParams.validate(params)
+
+        try:
+            client = TwilioService.get_client()
+
+            # Send SMS
+            client.messages.create(
+                to=params.recipient_phone,
+                messaging_service_sid=ConfigService.get_twilio_config(key="messaging_service_sid"),
```

Comment:
FYI - We are going to refactor config service in future tickets to just use getString, getInt etc for getting the value from config. It'd be nice if we can start right away and use/add something like - `ConfigService.get_string("twilio.messaging_service_sid")`

Reply:
I don't think it is possible for now as per the current declaration of `os_env.py` https://github.com/jalantechnologies/rflask-boilerplate/pull/52/files#diff-2e491804be8b23b220ca3329e69c6e2b17f39f83131c6e0294d266b448d72a3c 
As it is a dictionary we cannot use `dot` inside string to use it.

However if we want it to directly accessible by `get_string` function we have to give each variable a unique rather than dictionary to categorize it.
Let me know if we want to go with this method or let it be.